examples/Tutorial.ipynb

```diff
@@ -0,0 +1,925 @@
+    "\n",
+    "2. `select` defines how to select the candidate stars from the input catalog.  There are several ways to do this, but the easiest is some calculation based on stellar properties (columns from the input catalog).  There is also a size-magnitude selector which looks for the stellar locus in a size-magnitude diagram.  If this section is omitted, then all input objects that are not flagged are considered to be candidate stars.\n",
+    "\n",
+    "3. `psf` defines the PSF model.  This typically involves two parts: the model profile at any given location, and how that profile should be interpolated across the field of view.  This section may also define an outlier rejection scheme to remove bad fits from further consideration.\n",
+    "\n",
+    "4. `output` defines what file(s) to write to disk.  This may just be the Piff output file that defines the model (which other codes could then use to render PSF models), but it can also include additional plots and output catalogs in a `stats` field.  (More on that below.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4385ef7e",
+   "metadata": {},
+   "source": [
+    "All fields have a `type` (possibly implicit if there is a useful default), which governs what other parameters are expected/allowed in that field.  This closely matches onto Python classes and the other items are like initialization parameters.\n",
+    "\n",
+    "For instance, models may be any of the following types currently:\n",
+    "* Gaussian\n",
+    "* Moffat\n",
+    "* Kolmogorov\n",
+    "* PixelGrid\n",
+    "* Optical\n",
+    "\n",
+    "The currently implemented interpolation schemes are:\n",
+    "* Mean\n",
+    "* Polynomial\n",
+    "* BasisPolynomial\n",
+    "* kNNInterp\n",
+    "* GPInterp\n",
+    "\n",
+    "This structure makes all parts of the code very modular.  It is easy to add a new type for some part of the calculation, which a user can then specify in their config file to change the behavior of Piff for their use case."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5a6c0283",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Here is a relatively simple configuration for DES Y6 data:\n",
+    "\n",
+    "import yaml\n",
+    "\n",
+    "config = \"\"\"\n",
+    "\n",
+    "input:\n",
+    "    \n",
+    "    # dir is optional.  If set, it applies to all file names here without an absolute path\n",
+    "    dir: 572501\n",
+    "        \n",
+    "    # Define the input image with the pixel data.\n",
+    "    # These data come with a bad pixel mask and a weight map.\n",
```

Comment:
Is the weight map expected to be an inverse variance map? In which units? Should the variance also include the shot noise from the source? 
And for the bad-pixel map, is there any particular expected form? (e.g., "1" for good pixels, "0" for bad; or different mask planes like in the LSST Science Pipelines?)

Reply:
Yep.  LSST does it different from the usual kind of weight map.  They report variance, not inverse variance, and also include the signal as part of that variance, which we don't want.  So there are options to convert it.
```
invert_weight: True
remove_signal_from_weight: True
```

Reply:
I think I mentioned these verbally, but I'll add something about that in the tutorial.